frontend/src/components/MapView/LeftPanel/AnalysisPanel/index.tsx

```diff
@@ -486,21 +486,37 @@ function AnalysisPanel({
               {hazardDataType === GeometryType.Polygon && (
                 <>
                   <div className={classes.analysisPanelParamContainer}>
-                    <Typography variant="body2">Admin Level</Typography>
+                    <Typography
+                      className={classes.analysisParamTitle}
+                      variant="body2"
+                    >
+                      Admin Level
```

Comment:
While we're at it can we translate the parts that are not translated yet like this one?